dockers/docker-platform-monitor/Dockerfile.j2

```diff
@@ -26,7 +26,8 @@ RUN apt-get update &&   \
         psmisc          \
         python3-jsonschema \
         libpci3         \
-        iputils-ping
+        iputils-ping    \
+        smartmontools
```

Comment:
Please confirm that this will not conflict with the smartmontools installed on the host. 

Its also worth noting that `rules/smartmontools.mk` should have probably been cleaned up with https://github.com/sonic-net/sonic-buildimage/pull/10087 but it was not, now might be a good time to do so now that smartmontools is getting some attention.

Reply:
It might also be good practice to follow #10087 in hard-coding the version number to `smartmontools=7.2-1` so that we don't end up with mismatched versions. Again, not sure how particular its going to be about two instances of it running at the same time but it seems to me a version mismatch could exacerbate that.

Reply:
I've tested, and installing smartmontools in the docker does not conflict with the host.
As suggested, I've added the same version number to the docker and removed the $(SMARTMONTOOLS) files that remained.

Reply:
Both would have to be updated in the future. Unless there is a way for both to pick up the version from the same place (unsure if this is possible to be done easily), I can add a comment in both locations.

Reply:
@cytsao1 If this is not possible to only have one update and both are taken cared of, let's go with the "add comment in both locations" for now.
Thanks!

Reply:
This is relatively easy to do using Makefile variables in SONiC. In fact this is a great suggestion that I would like to see implemented. 

The rest of the changes LGTM. I'll approve once the above is resolved.

Reply:
@alexrallen if You can share a sample coding in the Makefile variable and @cytsao1 can give it a try?
Thanks so much!

Reply:
> Both would have to be updated in the future. Unless there is a way for both to pick up the version from the same place (unsure if this is possible to be done easily), I can add a comment in both locations.

In the interest of time, can you please go ahead with adding the comments on both locations so that at least in the future whoever need to upgrade it to future releases, both locations will be updated to avoid inconsistencies.

If you can figure out how to do this in the Make file to cover it with a variable that would be even better.

Reply:
I've added the comments for now.